test/ipynb/snapshot_tester.ipynb

```diff
@@ -3,35 +3,61 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "fc250437",
```

Comment:
It is possible to "clean" the notebook with `nb-clean`. In this way, we can keep the minimum file in the repo.

Reply:
Yeah, sure. I'll do that 😃